specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,1085 @@
+      "put": {
+        "tags": [
+          "StorageTasks"
+        ],
+        "description": "Asynchronously creates a new storage task resource with the specified parameters. If a storage task is already created and a subsequent create request is issued with different properties, the storage task properties will be updated. If a storage task is already created and a subsequent create or update request is issued with the exact same set of properties, the request will succeed.",
+        "operationId": "StorageTasks_Create",
+        "x-ms-examples": {
+          "PutStorageTask": {
+            "$ref": "./examples/storageTasksCrud/PutStorageTask.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StorageTaskName"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StorageTask"
+            },
+            "description": "The parameters to create a Storage Task."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- Update storage task with the same properties specified in the request body.",
+            "schema": {
+              "$ref": "#/definitions/StorageTask"
+            }
+          },
+          "201": {
+            "description": "OK -- Create storage task with the same properties specified in the request body.",
+            "schema": {
+              "$ref": "#/definitions/StorageTask"
+            }
+          },
+          "202": {
+            "description": "Accepted -- Create or update request accepted; operation will complete asynchronously.",
+            "headers": {
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "delete": {
+        "tags": [
+          "StorageTasks"
+        ],
+        "description": "Delete the storage task resource.",
+        "operationId": "StorageTasks_Delete",
+        "x-ms-examples": {
+          "PutStorageTask": {
```

Comment:
>PutStorageTask [](http://example.com/codeflow?start=11&length=14)

This should be named as DeleteStorageTask.

Reply:
+1 done